drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -728,7 +728,6 @@ class RelationshipToHoH < Types::BaseEnum
     value 'SPOUSE_OR_PARTNER', '(3) Spouse or partner', value: 3
     value 'OTHER_RELATIVE', '(4) Other relative', value: 4
     value 'UNRELATED_HOUSEHOLD_MEMBER', '(5) Unrelated household member', value: 5
-    value 'DATA_NOT_COLLECTED', '(99) Data not collected', value: 99
     value 'INVALID', 'Invalid Value', value: -999999
```

Comment:
An Enrollment with a missing or 99 value for Relationship To HoH will now resolve and appear as `Invalid` in the application. Is that what we want?